src/FormMigration/Steps/DoubleTheDonation.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Give\FormMigration\Steps;
+
+use Give\FormMigration\Contracts\FormMigrationStep;
+use Give\Framework\Blocks\BlockModel;
+
+/**
+ * @unreleased
+ */
+class DoubleTheDonation extends FormMigrationStep
+{
+
+    /**
+     * @unreleased
+     */
+    public function canHandle(): bool
+    {
+        return $this->formV2->getDoubleTheDonationStatus() === 'enabled';
+    }
+
+    /**
+     * @unreleased
+     */
+    public function process()
+    {
+        $block = BlockModel::make([
+            'name'       => 'givewp/dtd',
+            'attributes' => [
+                'label'   => $this->formV2->getDoubleTheDonationLabel(),
+                'company' => [
+                    'company_id'   => '',
+                    'company_name' => '',
+                    'entered_text' => '',
```

Comment:
@alaca is there any current information about these attributes in a v2 form? Meaning, if someone migrates from a v2 to v3 form with double the donation enabled, is there anyway to migrate this information as well (either through v2 form meta or global options) or is it new for v3 forms?

Reply:
This attribute is used as a placeholder so that I can set the values on frontend using the `setValue` hook. This information does not exist on the v2 or v3 form.